### DISCUSSION TITLE
browser block error

### DISCUSSION THREAD

**Discussion by kbarkhurst** [statamic/cms#5755] [Troubleshooting] [answered]
I went to extract a color palette using the Chrome extension https://palette.site on a Statamic site. It didn't work. A popup window reading example.com says Error with an "Okay" button happened instead. 

<img width="454" alt="Screen Shot 2022-04-05 at 11 39 29 AM" src="https://user-images.githubusercontent.com/7398062/161792154-780fb45a-2d5e-426a-9731-f664c697c1df.png">

I tested other websites, not on Statamic V2 or V3 CMS. They all work. Then I tested additional Statamic sites and as far as I can tell, by default, all Statamic sites do not allow this Chrome extension to work.

Curious mind here...
Why is this? What is it about the Statamic setup that is blocking this extension from working?

**Comment by duncanmcclean** [comment]
Are you in the Control Panel or the front-end of your site? Any failing network requests? Any console errors?

**Reply by kbarkhurst** [reply]
Thanks for jumping in, Duncan. I'm running the Site Palette Chrome Extension as a user on the public URLs on Statamic-built sites. Of interest, the extension fails on local builds of Statamic sites and public sites. The warning image I showed is from statamic.com. I opened Google Inspect and I'm not seeing any errors in the console. The only item of interest is this audit usage, which I don't think applies to this question

<img width="1177" alt="Screen Shot 2022-04-05 at 12 10 52 PM" src="https://user-images.githubusercontent.com/7398062/161798567-f7ea828e-24f9-4a2f-b1d8-bc659ae270a4.png">
.

**Comment by jasonvarga** [comment, marked as answer]
statamic.com is not a Statamic site.

I tried some other sites with that extension.
yahoo.com does absolute nothing.
The same error happens on duckduckgo.com

![image](https://user-images.githubusercontent.com/105211/161800949-d23aa6cb-ac13-400b-84a1-0036cac120a8.png)

I'd bring this up with the team for the chrome extension.

**Reply by kbarkhurst** [reply]
Thanks for looking into it, Jason. Appreciate it. This was really more of a curiosity about "how things work" more than anything else.

**Reply by jasonvarga** [reply]
My guess is that it injects some javascript into the page. Maybe there is some sort of conflict with existing javascript on certain sites.
